frontends/bas/package.json

```diff
@@ -15,8 +15,8 @@
     "lint:fix": "pnpm type-check && eslint . --fix && pnpm stylelint:run:fix",
     "start": "pnpm -w run-dev bas",
     "start:core": "pnpm -w run-dev bas --core-only",
-    "stylelint:run": "stylelint 'src/**/*.{js,jsx,ts,tsx}' && stylelint 'src/**/*.css' --config .stylelintrc-css.js",
-    "stylelint:run:fix": "stylelint 'src/**/*.{js,jsx,ts,tsx}' --fix && stylelint 'src/**/*.css' --config .stylelintrc-css.js --fix",
+    "stylelint:run": "stylelint 'src/**/*.{js,jsx,ts,tsx}'",
+    "stylelint:run:fix": "stylelint 'src/**/*.{js,jsx,ts,tsx}' --fix",
```

Comment:
I would also remove the `.stylelintrc-css.cs` files.

Reply:
FYI (not important to this PR, but you may hit it) is that the `--fix` option for `stylelint` is broken in `css-in-js`. There's not a good fix in the ecosystem yet, but it looks like one is evolving: https://github.com/styled-components/styled-components/issues/3607#issuecomment-1374344911. I'm tracking that thread and once something looks solid I'll make a ticket for anyone to complete.

Reply:
Ah, didn't know about the stylelint issue - thanks for tracking!

Also, good call re: .stylelintrc-css files